velox/functions/prestosql/aggregates/AverageAggregate.cpp

```diff
@@ -314,9 +314,10 @@ class DecimalAverageAggregate : public DecimalAggregate<TUnscaledType> {
   virtual TUnscaledType computeFinalValue(
       LongDecimalWithOverflowState* accumulator) final {
     // Handles round-up of fraction results.
+    int128_t sum = buildInt128(accumulator->upperSum, accumulator->lowerSum);
```

Comment:
Add a method in `LongDecimalWithOverflowState` to get/set the sum and handle conversion to/from int128 inside those methods.